rest/data/project_settings.go

```diff
@@ -91,8 +91,9 @@ func (sc *DBConnector) SaveProjectSettingsForSection(ctx context.Context, projec
 			// For repos, we need to use the repo ref functions, as they update different scopes/roles.
 			repoRef := &model.RepoRef{ProjectRef: *newProjectRef}
 			if err = repoRef.UpdateAdminRoles(adminsToAdd, adminsToDelete); err != nil {
-				return errors.Wrap(err, "error updating repo admin roles")
+				catcher.Wrap(err, "error updating repo admin roles")
 			}
+			newProjectRef.Admins = repoRef.Admins
```

Comment:
Now that we're potentially always updating, we should move modified = true above the if/else statement

Reply:
done